gatsby-node.ts

```diff
@@ -23,14 +24,28 @@ export const onCreateNode: GatsbyNode<SatellytesNode>['onCreateNode'] = async (
    * Provide a slug for any markdown page (which is not given by default)
    * to build our pages based on this markdown file.
    */
-  if (node.internal.type === `MarkdownRemark`) {
+  if (
+    node.internal.type === `MarkdownRemark` &&
+    Object.keys(node).includes('fileAbsolutePath')
```

Comment:
Is it because the Markdown types in a Contentful object don't have an absolute path? A comment above would be helpful here.

Reply:
@feedm3 Yes, your assumption is correct. Added a small note.